.eslintrc.js

```diff
@@ -5,14 +5,15 @@ module.exports = {
     'node_modules',
     '!.*.js',
     'packages/analytics',
+    'packages/codemods',
     'packages/common',
     'packages/design-tokens',
+    'packages/form-validations',
+    'packages/icons',
     'packages/web-react',
     'packages/web',
-    'packages/form-validations',
-    'packages/codemods',
-    'exporters/scss',
     'exporters/js',
+    'exporters/scss',
     'exporters/tokens',
     'examples/*'
```

Comment:
Nitpicking: If order doesn't matter and we want to sort pattern 🔤, shouldn't it be something like this?

```
    '!.*.js',
    'examples/*'
    'exporters/js',
    'exporters/scss',
    'exporters/tokens',
    'node_modules',
    'packages/analytics',
    'packages/codemods',
    'packages/common',
    'packages/design-tokens',
    'packages/form-validations',
    'packages/icons',
    'packages/web',
    'packages/web-react',
```